test/auth.js

```diff
@@ -9,9 +9,9 @@ const Account = require("../src/models/Account");
 
 describe("Register endpoint test", () => {
 
-    afterEach("Clearing accounts", async () => {
-        await Account.deleteMany({});
-    });
+    before("Clearing accounts", () => Account.deleteMany({}));
+    after("Clearing accounts", () => Account.deleteMany({}));
```

Comment:
The basis for the change here is that the methods themselves return a `Promise` and this will make the hooks await the promise, correct?

Despite this, I'm not sure if this is very clear. I feel that this might result in someone adding braces to the arrow function and then not knowing that an `await` is necessary.

*TL;DR*: Explicit > Implicit, it is better to handle promises explicitly than "hiding" them like this.
But this can use some other feedback, it is open for discussion.

Reply:
The objective is to return a Promise, if I understood correctly the suggestion is to include bracers and the return keyword, however eslint is configured so that this is not possible.

Reply:
I know that the objective is to return a promise 😅 I might not have been very clear, I'll try to explain it better:

Returning a Promise in a test hook makes the test execution wait for the hook completion.
However, returning a Promise there is being implicit behaviour (due to how arrow functions work).

I get what you mean about the `eslint` rules, but that was not what I had in mind.
I was thinking of something like:

```js
before("Hook", async () => {
    await Async.stuff();
});
```

Due to the way that `async` methods work, that is functionally the same as returning a `Promise`.
However, if done this way, it will be clear that the hook will wait for the async stuff to resolve, because it is more explicit.

My recommendation is to switch to this style in order to make sure that the code is clearer for other people that might look at it and reduce possible copy paste errors that might occurr (someone might copy paste the function and add braces and more statements, and then the promise would not be returned anymore - thus making the methods in the hook not be `await`ed for).

Reply:
This can (and should, imo) also be changed in tests, that I see are following a similar pattern.  This would also make them more consistent with what's being done in #38 😄